### ISSUE TITLE
JS Only example, or build config example?

### ISSUE THREAD

**Issue by kevinmpowell** [Boulevard/vampire#4]
@daniel-nagy great work on this. I really want to use it in combination with Lit Element but I'm struggling to figure out how to integrate it into my existing set of components. I'm using a very simple rollup config with only `typescript` and `resolve` plugins. But I'm getting other errors related to the `@babel/plugin-proposal-class-properties` plugin.
 
I don't want to use babel for my modern browser build if possible, but I'm unclear on how to get it all wired up.

Any thoughts or clarity you could add to the README would be helpful. Thanks!

**Comment by daniel-nagy** [comment]
Are you importing the module for [side effects](https://developer.mozilla.org/en-US/docs/web/javascript/reference/statements/import#Import_a_module_for_its_side_effects_only)?

```typescript
import '@boulevard/vampire';
```

Rollup may be tree shaking the exports because it doesn't know that they contain side effects. Can you confirm this by looking at your rollup output and seeing if it contains any logic from this [file](https://github.com/Boulevard/vampire/blob/master/src/vampire-root.ts).

> I'm using a very simple rollup config with only typescript and resolve plugins. But I'm getting other errors related to the @babel/plugin-proposal-class-properties plugin.

This module wouldn't have anything to do with that, it sounds like you have some issues with your rollup config.

**Comment by kevinmpowell** [comment]
> Are you importing the module for [side effects](https://developer.mozilla.org/en-US/docs/web/javascript/reference/statements/import#Import_a_module_for_its_side_effects_only)?

Yes. My import is the same. All of the VampireRoot file exists in the compiled output. 

@daniel-nagy would it be possible for you to share your compiler config? The stackblitz example in the readme seems to obscure how the code is actually being compiled.

Edit: And I'm not super familiar with Stackblitz's interface, it's possible there's a config view somewhere that I don't know how to access.
